### DISCUSSION TITLE
Add_checkpoint step is throwing "NodeEvent/DocumentStartEvent" and "I/O operation on closed file" errors

### DISCUSSION THREAD

**Discussion by staggjim** [great-expectations/great_expectations#6025] [Support]
I am running GE expectations, in Python, but currently I am getting errors returned from the add_checkpoint step in my code. Initially I get the below error:

`Exception: Expectations for table d_pharmacy failed! expected NodeEvent, but got DocumentStartEvent() `

And then on subsequent runs I get the following error:

`Exception: Expectations for table d_pharmacy failed! I/O operation on closed file \n {\"$type\":\"System.Exception, System.Private.CoreLib\",\"ClassName\":\"System.Exception\",\"Message\":\" Exception: Expectations for table d_pharmacy failed! I/O operation on closed file\`

From testing I have isolated the error to the add_checkpoint step. This is my config code, can you advise if I have anything missing or incorrect in my config that would be causing these errors?

```
def run_checkpoint(table_name, context, batch_request, expectation_suite_name):
    my_checkpoint_name = f"{table_name}_checkpoint"
    checkpoint_config = {
        "name": my_checkpoint_name,
        "config_version":  1.0,
        "class_name": "SimpleCheckpoint",
        "run_name_template": "%Y%m%d-%H%M%S-emisar-template",
    }

    # my_checkpoint = context.test_yaml_config(yaml.dump(checkpoint_config))
    context.add_checkpoint(**checkpoint_config)
```

**Comment by kyleaton** [comment]
Hi @staggjim, 

It seems we're searching for an answer on our slack for this issue with you. Here's a link for others to follow along and when we reach a conclusion we'll post it into this discussion. 

https://greatexpectationstalk.slack.com/archives/CUTCNHN82/p1660837115529949

Thanks,
Kyle
